src/search-schema/data/definitions/enums/assay_types.yaml

```diff
@@ -22,72 +26,86 @@ bulk_atacseq:
 #    alt-names: [bulk-atacseq]
     alt-names: []
     primary: false
+    contains-pii: false
     vitessce-hints: []
 
 cell-dive:
     description: Cell DIVE
     alt-names: []
     primary: true
+    contains-pii: false
 
 CODEX:
     description: CODEX
     alt-names: []
     primary: true
+    contains-pii: false
     vitessce-hints: []
 
 codex_cytokit:
     description: CODEX [Cytokit + SPRM]
     alt-names: []
     primary: false
+    contains-pii: false
     vitessce-hints: ['codex', 'is_image', 'is_tiled']
 
 DART-FISH:
     description: DART-FISH
     alt-names: []
     primary: true
     vitessce-hints: []
+    contains-pii: false
 
 image_pyramid:
     description: Image Pyramid
     alt-names: []
     primary: false
+    vis-only: true
```

Comment:
Why do some of these contain vis-only field and others do not? assuming types marked vis-only=true are ONLY used for visualization.

Reply:
That's correct @ropelews .  The vis-ony key defaults to false, just as the contains-pii key defaults to true.